File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_ones = 0
        curr_ones = 0
        for i in range(len(nums)):
            if nums[i] == 1:
               curr_ones += 1
            else:
                if max_ones < curr_ones:
                    max_ones = curr_ones
                curr_ones = 0
        if max_ones < curr_ones:
            max_ones = curr_ones
        return max_ones 
        

File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int len = nums.length;
        int l = 0;
        int r = 0;

        while(r < len) {
            while(r < len && nums[r] != val) {
                nums[l] = nums[r];
                l++;
                r++;
            }
            r++;
            
            
        }

        return l;
    }
}